protostar/commands/test/testing_summary.py

```diff
@@ -183,7 +188,9 @@ def _format_slow_test_cases_list(
         local_log_color_provider: LogColorProvider = log_color_provider,
     ) -> str:
 
-        slowest_test_cases = self._get_slowest_test_cases_list(self.failed + self.passed, count)  # type: ignore
+        slowest_test_cases = self._get_slowest_test_cases_list(
+            self.failed + self.passed + self.broken, count  # type: ignore
```

Comment:
```suggestion
            self.case_results, count
```

doesn't this work?

Reply:
No, because case_results also stores BrokenTestSuites. We might consider changing it though.

Reply:
I actually wonder why this method accepts a list of test cases as argument, given that it knows all results due to being a method of the summary class. Maybe we should simply remove this argument and filter case_results by isinstance internally?

Reply:
Oh, I think it's a leftover from a time when this method was in a different file.